apps/dashboard/src/components/auth/mobile-message.tsx

```diff
@@ -0,0 +1,57 @@
+import { post } from '@/api/api.client';
+import { Smartphone } from 'lucide-react';
+import { useEffect } from 'react';
+import { showErrorToast } from '../primitives/sonner-helpers';
+
+const MOBILE_WIDTH_THRESHOLD = 768;
+const FIVE_MINUTES_MS = 5 * 60 * 1000;
+const MOBILE_SETUP_STORAGE_KEY = 'mobileSetupEmailSentAt';
+
+export function MobileMessage() {
+  useEffect(() => {
+    const notifyMobileSetup = async () => {
+      try {
+        const isMobile = window.innerWidth < MOBILE_WIDTH_THRESHOLD;
+        const lastSentAt = localStorage.getItem(MOBILE_SETUP_STORAGE_KEY);
+
+        const now = Date.now();
+        const shouldSendEmail = !lastSentAt || now - parseInt(lastSentAt) > FIVE_MINUTES_MS;
```

Comment:
Just want to throttle this a bit in case the user will reload this page

Reply:
what do you think about changing it to a longer duration, feels like one email in a day could be enough.